docs/src/main/paradox/getting-started/running-cluster.md

```diff
@@ -0,0 +1,95 @@
+[2020-08-13 15:03:39,812] [INFO] [docs.guide.ItemPopularityProjectionHandler] [] [ShoppingCartClusterApp-akka.actor.default-dispatcher-43] - ItemPopularityProjectionHandler(shopping-cart-0) item popularity for 'akka t-shirt': [1080] MDC: {}
+...
+```
+
+Run a second member in a new terminal to expand the akka cluster member count to 2.
+
+<!-- run from repo:
+sbt "examples/test:runMain docs.guide.ShoppingCartClusterApp 2552"
+-->
+```shell
+sbt "runMain docs.guide.ShoppingCartClusterApp 2552"
+```
+
+When the second app is running you will observe a sharding rebalance complete and then see a distinct set of tagged events processed on each cluster member.
+
+A rebalance occurs and tag `shopping-cart-0` is assigned to the new cluster member. 
+Only tags `shopping-cart-1` and `shopping-cart-2` are processed by the first member.
+
+```
+[2020-08-13 15:03:59,019] [INFO] [akka.cluster.sharding.DDataShardCoordinator] [] [ShoppingCartClusterApp-akka.actor.default-dispatcher-41] - Starting rebalance for shards [0]. Current shards rebalancing: [] MDC: {akkaAddress=akka://ShoppingCa
+rtClusterApp@127.0.0.1:2551, sourceThread=ShoppingCartClusterApp-akka.actor.default-dispatcher-44, akkaSource=akka://ShoppingCartClusterApp@127.0.0.1:2551/system/sharding/sharded-daemon-process-shopping-cartsCoordinator/singleton/coordinator, 
+sourceActorSystem=ShoppingCartClusterApp, akkaTimestamp=19:03:59.019UTC}                                                                                                                                                                           
+[2020-08-13 15:04:35,261] [INFO] [docs.guide.ItemPopularityProjectionHandler] [] [ShoppingCartClusterApp-akka.actor.default-dispatcher-43] - ItemPopularityProjectionHandler(shopping-cart-1) item popularity for 'skis': [1244] MDC: {}           
+[2020-08-13 15:04:36,802] [INFO] [docs.guide.ItemPopularityProjectionHandler] [] [ShoppingCartClusterApp-akka.actor.default-dispatcher-40] - ItemPopularityProjectionHandler(shopping-cart-2) item popularity for 'skis': [1246] MDC: {}           
+[2020-08-13 15:04:36,805] [INFO] [docs.guide.ItemPopularityProjectionHandler] [] [ShoppingCartClusterApp-akka.actor.default-dispatcher-40] - ItemPopularityProjectionHandler(shopping-cart-2) item popularity for 'akka t-shirt': [1136] MDC: {}   
+[2020-08-13 15:04:36,807] [INFO] [docs.guide.ItemPopularityProjectionHandler] [] [ShoppingCartClusterApp-akka.actor.default-dispatcher-43] - ItemPopularityProjectionHandler(shopping-cart-2) item popularity for 'skis': [1249] MDC: {}           
+[2020-08-13 15:04:39,262] [INFO] [docs.guide.ItemPopularityProjectionHandler] [] [ShoppingCartClusterApp-akka.actor.default-dispatcher-41] - ItemPopularityProjectionHandler(shopping-cart-1) item popularity for 'cat t-shirt': [1239] MDC: {}                  
+...
+```
+
+When the second member joins the cluster it is assigned tag `shopping-cart-0` and begins processing events.
```

Comment:
maybe mention that it will start from saved offset so it will only process new events

Reply:
:+1: